api/create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE IF NOT EXISTS cs348db;
+USE cs348db;
+
+CREATE TABLE IF NOT EXISTS UnitListerAccount(
+    pm_id int NOT NULL AUTO_INCREMENT,  
+    name VARCHAR(255) NOT NULL,
+    phone_num VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    website VARCHAR(255) NOT NULL,
+    PRIMARY KEY (pm_id)
+    );
+
+CREATE TABLE IF NOT EXISTS Building(
+    building_id int NOT NULL AUTO_INCREMENT,
+    address VARCHAR(255) NOT NULL,
+    pet_friendly TINYINT(1) NOT NULL, -- TINYINT functions as a boolean
+    laundry_availability VARCHAR(255) NOT NULL, -- Do we want a yes or no, or ensuite vs in building
+    type_of_unit VARCHAR(255) NOT NULL, -- use a multiselect on frontend so values are consistent
+    distance_from_waterloo DECIMAL(3,1) NOT NULL,
+    PRIMARY KEY (building_id)
+);
+
+CREATE TABLE IF NOT EXISTS AvailableUnit(
+    unit_id int NOT NULL AUTO_INCREMENT,
+    building_id int,
+    pm_id int,
```

Comment:
This should be NOT NULL, since units can't be posted without an account
```suggestion
    pm_id int NOT NULL,
```